src/webui/api/torrentscontroller.cpp

```diff
@@ -787,7 +785,6 @@ void TorrentsController::pieceStatesAction()
 void TorrentsController::addAction()
 {
     const QString urls = params()[u"urls"_s];
-    const QString cookie = params()[u"cookie"_s];
```

Comment:
Do we have an API bump yet in 5.1? It'd be nice to get this in 5.0 if possible but otherwise can bump the WebAPI in 5.1 too.